### ISSUE TITLE
Number of seconds to next quarter/hour calculated wrongly

### ISSUE THREAD

**Issue by sander48k** [hozn/stravalib#155]
The `get_seconds_until_next_quarter` and `get_seconds_until_next_day` functions in `stravalib.util.limiter` always return the same value when called without a parameter. The default parameter (now=arrow.utcnow()) is evaluated only once when the function definition is executed (see [here](https://docs.python.org/2.0/ref/function.html).

**Comment by hozn** [comment]
Yeah, that wouldn't work at all :-)  I'm embarrassed that I merged that in.

**Comment by sander48k** [comment]
Haha, don't be! It's really a great library overall. Works perfectly for my application. Keep up the good work!

**Comment by jsamoocha** [comment]
My bad, I'll take a look...

**Comment by hozn** [comment]
@jsamoocha , no worries; i should have caught it.   this pattern would be best:
```python
def myfunc(now = None):
  if now is None:
    now = arrow.utcnow()
```

**Comment by jsamoocha** [comment]
That may break some unit tests
